Proper PR with the changes requested

greed2411 opened the pull request:
This PR contains the changes mentioned in [report](https://github.com/Jaiimmortal/biosql.github.io/blob/master/report_of_changes.md), there were 13 broken links and all were found and replaced using Google, and [biosql](https://github.com/biosql/biosql), [biojava](https://github.com/biojava/biojava), [biojava-legacy](https://github.com/biojava/biojava-legacy) as requested by @peterjc , I hope I did it properly this time, if not mention, I would do it right away.

For the issue: https://github.com/biosql/biosql.github.io/issues/6

greed2411 committed:
Add files via upload

Dealing the issues #6 and #4

greed2411 committed:
Updated the broken link for J. Celko, SQL for Smarties

greed2411 committed:
Actual report , previous one was faulty

greed2411 committed:
Update results.md

greed2411 committed:
Update BioJava_BioSQL_ORM.md

greed2411 committed:
Update results.md

greed2411 committed:
Update BioJava_BioSQL_ORM.md

greed2411 committed:
Update results.md

greed2411 committed:
Update results.md

greed2411 committed:
Update results.md

greed2411 committed:
Update results.md

greed2411 committed:
Update results.md

greed2411 committed:
Update results.md

greed2411 committed:
Update Enhancement_Requests.md

greed2411 committed:
Update Enhancement_Requests.md

greed2411 committed:
Update Enhancement_Requests.md

greed2411 committed:
Update Enhancement_Requests.md

greed2411 committed:
Update results.md

greed2411 committed:
Update results.md

greed2411 committed:
Update results.md

greed2411 committed:
Update results.md

greed2411 committed:
Update results.md

greed2411 committed:
Update Releases.md

greed2411 committed:
Update Main_Page.md

greed2411 committed:
Update and rename results.md to report_of_changes.md

peterjc commented:
Can you remove the ``results.md`` aka ``report_of_changes.md`` file from the pull request please?

peterjc commented on wiki/Enhancement_Requests.md:
```diff
@@ -282,7 +282,7 @@ would only consist of gap characters, and be virtual).
 
 Related to CO lines, [EMBLs AS
-lines](http://www.ebi.ac.uk/embl/Documentation/User_manual/usrman.html#3_4_13)
+lines]<http://ftp.ebi.ac.uk/pub/databases/embl/doc/usrman.txt#3_4_13>
```
These should be round brackets not the less than/greater than signs (for markdown links).

peterjc commented on wiki/PhyloDBSchema.md:
```diff
@@ -82,7 +82,7 @@ part of. "left\_idx and "right\_idx" specify the left and right values
 queries. These values needs to be precomputed by a program, see [J.
 Celko, SQL for
-Smarties](http://rs18.rapidshare.com/files/4441805/Trees_and_Hierarchies_in_SQL_for_Smarties.pdf).
+Smarties](https://datubaze.files.wordpress.com/2016/03/celkos-trees-and-hierarchies-in-sql_2004.pdf).
```
@hlapp I'm not sure if we should be linking to an Elsevier copyright book here - would a reference perhaps with ISBN 1-55860-920-2 be better?

hlapp replied:
I agree, the PDF is probably unintentionally public, and is fragile as it's subject to a DMCA takedown notice from the copyright holder. I'd suggest to use the ISBN link, here's the one through Worldcat: https://www.worldcat.org/search?q=isbn%3A0128007613

greed2411 replied:
Done, Sir.

peterjc commented:
I noticed one link markup problem, and queried linking directly to a copyright book, but otherwise this looks good. Thanks for working on this.

greed2411 committed:
Update Enhancement_Requests.md

greed2411 committed:
Delete report_of_changes.md

greed2411 commented:
I did the changes you asked for Sir, , I removed the report_of_changes.md and changed <> to ()

As of rights to use that link for the book, I can't say anything you guys decide.

I've got one humble request to ask you @peterjc I know it's too much to ask, but this is my first contribution to open source and you guys have directed me in the right way, and I can't thank enough, I've started pursuing bioinformatics and going to be a sophomore, Is it okay to ask to be a part of your organization @biosql, I just did the documentation, but sure can help you guys a lot in the near future. What do you say?

Thanks, for the opportunity.

peterjc commented:
BioSQL is not currently a very active project, so it is lucky to have you making your first open source project contributions here:

https://github.com/biosql/biosql/graphs/contributors
https://github.com/biosql/biosql.github.io/graphs/contributors

If this and SQL is something you are interested in, then perhaps you might like to work on the programming language specific bindings for BioSQL in Perl, Python, Ruby or Java, under the BioPerl, Biopython, BioRuby or BioJava projects respectively?

However, there are hundreds of open source bioinformatics tools and projects, so if you approach learning new tools with this kind of positive attitude you are bound to be able to make other useful contributions - probably starting with documentation fixes, but later code bug fixes, etc.

I'm going to leave further review of this pull request to Hilmar (BioSQL project lead) to approve.

greed2411 commented:
Yes thanks a lot, I'll try to make my learning curve steeper in the upcoming days and try to contribute as much as possible to you guys and other Open Source projects. 

Nonetheless, thanks for this opportunity @peterjc for being supportive throughout the PR and @hlapp  for BioSQL.

greed2411 committed:
Update PhyloDBSchema.md

hlapp commented:
> Is it okay to ask to be a part of your organization @biosql, I just did the documentation, but sure can help you guys a lot in the near future. What do you say?

Hi @Jaiimmortal - as @peterjc's comment, with your contribution you've become part of the Biosql contributors already 😄  But if you're interested in engaging more with the wider community that the Biosql project is a part of, you might want to check out the [Open Bioinformatics Foundation (OBF)](http://open-bio.org). OBF does [maintain a membership](https://www.open-bio.org/wiki/Membership) to which you can sign up if you haven't already.   And it'd be great to see you at a [BOSC](http://open-bio.org/wiki/BOSC) conference.

greed2411 commented:
Ooh thanks for that information, and I can't express my gratitude properly @hlapp , I'll stick around and start helping out ppl, and thanks for the invite but I don't think I can attend the conference, I'm from India.
